clients/mobile/src/LocationService/LocationService.ts

```diff
@@ -40,4 +63,84 @@ export class LocationService {
       }
     }
   }
+
+  async preferedLocationAutocomplete(location: string, city: string): Promise<Array<{ label: string; value: string }>> {
+    const COUNTRY = 'BD'
```

Comment:
should this const be here ? shouldn't it be moved to somewhere else like a noMagicValues file or something?
also is this really important that we hardcode the country? given we can just instantly know where the user is currently? and use that data? and use this as a fallback?

Reply:
Move the country to the environment configuration. As you mentioned, we can determine the user's location by retrieving it from their device. However, what happens to the city? The city is hardcoded in our codebase, and we are not fetching it dynamically from any source. We are selecting cities that belong to Bangladesh (BD), but we are looking for locations or health centers that are not in Bangladesh.